kinto/tests/test_views_collections.py

```diff
@@ -107,3 +103,26 @@ def test_records_of_collection_are_deleted_too(self):
         resp = self.app.get('%s/records?_since=0' % self.collection_url,
                             headers=self.headers)
         self.assertEqual(len(resp.json['data']), 0)
+
+
+class CollectionCreationTest(BaseWebTest, unittest.TestCase):
+
+    collections_url = '/buckets/beers/collections'
+
+    def setUp(self):
+        super(CollectionCreationTest, self).setUp()
+        self.app.put_json('/buckets/beers', MINIMALIST_BUCKET,
+                          headers=self.headers)
+
+    def test_collection_can_be_created_with_post(self):
+        r = self.app.post_json(self.collections_url,
+                               MINIMALIST_COLLECTION,
+                               headers=self.headers)
+        self.assertEqual(r.status_code, 201)
```

Comment:
You mean here to check that ``id`` returned is not empty? Next tests checks explicitly the id.